requirements.txt

```
@@ -1,6 +1,6 @@
-Flask==2.3.3
-prometheus-client==0.17.1
-python-dateutil==2.8.2
-pytz==2023.3.post1
+Flask==3.1.0
+prometheus-client==0.21.1
+python-dateutil==2.9.0
+pytz==2025.1
 PyYAML==6.0.2
```

```suggestion
PyYAML==6.0.2
setuptools==75.8.2
```
Locally I cannot build it without directly specifying to add setuptools. I guess it was a transitive dependency in the past, specifying it directly is a good idea anyway.

---

Fixed